flocker_bb/builders/flocker.py

```diff
@@ -348,36 +338,36 @@ def makeInternalDocsFactory():
             '--verbose',
             '--delete-removed',
             '--no-preserve',
-            Interpolate('%s/%s/docs/' % (branch, revision)),
+            resultPath('docs'),
             Interpolate(
                 "s3://%(kw:bucket)s/%(prop:version)s/",
                 bucket='clusterhq-dev-docs',
             ),
         ],
-        path="private_html",
         doStepIf=isReleaseBranch('flocker'),
     ))
     return factory
 
 
-def createRepository(distribution):
+def createRepository(distribution, repository_path):
     steps = []
     flavour, version = distribution.split('-', 1)
     if flavour in ("fedora", "centos"):
-        steps.append(ShellCommand(
+        steps.append(MasterShellCommand(
```

Comment:
All results are deleted after 14 days ([here](https://github.com/ClusterHQ/build.clusterhq.com/blob/master/flocker_bb/builders/maint.py#L16-L22)). The next run of this will remove those files from the generated metadata.